CONTRIBUTING.md

```diff
@@ -95,6 +96,98 @@ have more control over the process and debug any issues.
 1. Run `rush change` to generate changelog. You will need to follow the interactive prompts.
    You can edit the added files later. If you don't add the right entries, the CI build will fail.
 
+# How to add and roll out new linter rules
```

Comment:
>How [](http://example.com/codeflow?start=2&length=3)

Lets have the criteria for checking the rule against Typespec generated specs show up here. We may not have a way right away using kusto to get this but we should atleast manually check this on github before promoting